README.rst

```diff
@@ -151,6 +150,27 @@ dynamic programming and is linear O(n):
     >>> print(big_o.big_o(fib_dp, big_o.datagen.n_, n_repeats=100, min_n=200, max_n=1000)[0])
     Linear: time = -1.8E-06 + 7.3E-06*n (sec)
 
+Report Generation
+-------
+
+This feature allows users to generate a report based on the outputs received from
+calling the :code:`big-o` function.
+The report defines the best time complexity along with the the others
+estimates and returns them as a string.
+
+    >>> best, others = big_o.big_o(heapify, data_generator_heapify, max_n=10**7)
+    >>> print(big_o.reports.big_o_report(best, others))
+    Best : Polynomial: time = 3.5E-06 * x^0.97 (sec)
+    Constant: time = 0.13 (sec)                                     (res: 0.067)
+    Linear: time = 0.0068 + 2.5E-06*n (sec)                         (res: 0.003)
+    Quadratic: time = 0.053 + 2.2E-11*n^2 (sec)                     (res: 0.012)
+    Cubic: time = 0.074 + 2.1E-16*n^3 (sec)                         (res: 0.02)
+    Polynomial: time = 3.5E-06 * x^0.97 (sec)                       (res: 0.003)
+    Logarithmic: time = -0.2 + 0.033*log(n) (sec)                   (res: 0.027)
+    Linearithmic: time = 0.013 + 2.2E-07*n*log(n) (sec)             (res: 0.0035)
+    Exponential: time = 0.007 * 1^n (sec)                           (res: 0.22)
+    measures:                                                    [   100  11200  22300  33400  44500  55600  66700  77800  88900 100000]
+    times:                                                       [0.00031296 0.02925126 0.06087168 0.08566567 0.10440871 0.18683418 0.19720671 0.2052643  0.21313027 0.24182967]
```

Comment:
The report function below does not print "measures" or "times"

Reply:
Do you want me to remove the measures and times field even if it is passed as a part of the input ?

Reply:
I don't think `measures` and `times` are part of the output you would get from `print(big_o.reports.big_o_report(best, others))`. Or are they?

Reply:
Yes they are returned when return_raw_data=True. 
Try Running this `best, others = big_o.big_o(sorted, lambda n: big_o.datagen.integers(n, 100, 500), return_raw_data=True)`

Reply:
I can sure remove them, If it's not needed in the report

Reply:
ah I understand now. still, in this example `big_o` is not called with `return_raw_data=True` so they woudln't be printed

Reply:
Oh yea! My bad. Will update that

Reply:
Made the needed changes!